cypress/support/commands.ts

```diff
@@ -95,13 +94,17 @@ export const haveAriaLabel = (text: string) => ($target: JQuery) => {
   }
 };
 
+const isKeyOf = <T>(obj: T, key: any): key is keyof T => {
+  return typeof key === 'string' && key in obj;
+};
+
 Cypress.Commands.overwrite('should', (originalFn, subject, expectation, ...args) => {
   const customMatchers = {
     'have.ariaDescription': haveAriaDescription(args[0]),
     'have.ariaLabel': haveAriaLabel(args[0]),
   };
   // See if the expectation is a string and if it is a member of Jest's expect
-  if (typeof expectation === 'string' && customMatchers[expectation]) {
+  if (isKeyOf(customMatchers, expectation)) {
```

Comment:
I enabled strict Typescript checking on the cypress directory. This helps tell typescript that `expectation` is a valid key of `customMatchers`